package.json

```diff
@@ -1044,13 +1044,24 @@
     "@babel/plugin-transform-runtime": "^7.0.0",
     "@babel/preset-env": "^7.0.0",
     "@babel/preset-react": "^7.9.0",
+    "@folio/eslint-config-stripes": "^7.0.0",
+    "@folio/jest-config-stripes": "^2.0.0",
+    "@folio/service-interaction": "^4.0.0",
+    "@folio/stripes": "^9.2.2",
+    "@folio/stripes-cli": "^3.2.0",
+    "@folio/stripes-components": "^12.2.2",
+    "@folio/stripes-connect": "^9.2.0",
+    "@folio/stripes-core": "^10.2.1",
+    "@folio/stripes-erm-components": "^10.0.0",
```

Comment:
It looks like old dependencies are added.

Reply:
Thank you @Dmytro-Melnyshyn !
Sorry, that was a mistake while merging.